app/models/task.rb

```diff
@@ -0,0 +1,23 @@
+class Task < ActiveRecord::Base
+
+  belongs_to :person
+
+  STATUS = {is_complete: ["uncomplete", "complete"], not_complete: ["complete", "uncomplete"]}
+
+  def self.all_tasks
```

Comment:
Haha I think I wrote that the first day, before I knew about the .all that active record provides. I should have removed it!